### PR TITLE
Added T1 EWAR Cruisers

### PR DESCRIPTION
**Pull request by Epicloa-Program**
Fixed slight formatting error on T1 Frigates, shouldn't change anything.

### CODE REVIEW

File: Fits/T1 EWAR Cruisers/[NVY-30] Arbitrator.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+Tracking Disruptor - Use against ships with Guns/Turrets.
+Guidance Disruptor - Use against ships with Missiles.
+Range Disruption - Use against ships with long range capabilities.
+Tracking Speed / Precision Disruption - Use against bigger ships shooting smaller ships. 
+
+## Fit
+
+```
+[Arbitrator, [NVY-30] Arbitrator]
```
Comment (bearthatcares) [inline Fits/T1 EWAR Cruisers/[NVY-30] Arbitrator.md:17]: Better as a brick IMO, can actually stay alive
```
[Arbitrator, [NVY-30] Arbitrator]

Damage Control II
1600mm Crystalline Carbonide Restrained Plates
Multispectrum Energized Membrane II
Multispectrum Energized Membrane II
Reactive Armor Hardener

50MN Quad LiF Restrained Microwarpdrive
DDO Scoped Tracking Disruptor I
DDO Scoped Tracking Disruptor I
DDO Scoped Tracking Disruptor I

Drone Link Augmentor I
Drone Link Augmentor I
Auto Targeting System I
[Empty High slot]

Medium Trimark Armor Pump I
Medium Trimark Armor Pump I
Medium Trimark Armor Pump I


Acolyte I x7
Hornet EC-300 x7
Vespa EC-600 x8


Missile Precision Disruption Script x3
Missile Range Disruption Script x3
Nanite Repair Paste x10
Optimal Range Disruption Script x3
Tracking Speed Disruption Script x3
Mobile Depot x1
Highstroke Scoped Guidance Disruptor x3
```

File: Fits/T1 EWAR Cruisers/[NVY-30] Blackbird.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+Ladar - Minmatar, Angel Cartel.
+Radar - Amarr, Blood Raiders, Sansha's Nation, Triglavian Collective.
+Magnometric - Gallente, ORE, Sisters of Eve.
+Gravimetric - Caldari, Guristas, Mordu's Legion.
+
+## Fit
+
+```
+[Blackbird, [NVY-30] Blackbird]
```
Comment (bearthatcares) [inline Fits/T1 EWAR Cruisers/[NVY-30] Blackbird.md:19]: Not really worth it as an armor ship, immediately gets deleted, doesn't have the tank like a Scorpion. Griffin is better if you fit Blackbird like this.  Can be decent as a shield brick, though.

```
[Blackbird, [ADV-30] Blackbird]

Damage Control II
Mark I Compact Power Diagnostic System
Mark I Compact Power Diagnostic System

50MN Quad LiF Restrained Microwarpdrive
Enduring Multispectrum Shield Hardener
Enduring Multispectrum Shield Hardener
Large F-S9 Regolith Compact Shield Extender
Large F-S9 Regolith Compact Shield Extender
Deluge Enduring Burst Jammer

[Empty High slot]
[Empty High slot]
[Empty High slot]
[Empty High slot]

Medium Core Defense Field Extender I
Medium Core Defense Field Extender I
Medium Core Defense Field Extender I


FZ-3a Enduring Gravimetric ECM x1
Gloom Enduring Radar ECM x1
Languid Enduring Ladar ECM x1
Morpheus Enduring Magnetometric ECM x1
```

File: Fits/T1 EWAR Cruisers/[NVY-30] Celestis.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+Utilizes Remote Sensor Dampeners (Damps) to disrupt hostile DPS or Logistics. This ship is entirely reliant on range to avoid incoming DPS so do not burn close unless absolutely necessary.
+
+Targeting Range - Use on long range platforms like enemy ECM or ships with long engagement ranges.
+Scan Resolution - Use on Logistics ships that are close to the enemy fleet or against any ship that is too close for targeting range to make a difference.
+
+## Fit
+
+```
+[Celestis, [NVY-30] Celestis]
```
Comment (bearthatcares) [inline Fits/T1 EWAR Cruisers/[NVY-30] Celestis.md:15]: Similar to Arbitrator, this ship can actually survive if you fit it like a brick.

File: Fits/T1 EWAR Cruisers/[NVY-30] Arbitrator.md (hunk 1 above)
Comment (Epicloa-Program) [reply Fits/T1 EWAR Cruisers/[NVY-30] Arbitrator.md:17]: The primary issue is that yeah you gain a a bit of raw EHP but you're also forcing yourself to actually be in range to take damage. That cuts the effective range of the fit from 135k down to 87.5k and that range is always going to be a better tank than a T1 cruiser's EHP.

**Pull request merged**